test/builtin/arithmetic/test_basic.py

```diff
@@ -164,7 +177,7 @@ def test_multiply(str_expr, str_expected, msg):
         ("I^(2/3)", "(-1) ^ (1 / 3)", None),
         # In WMA, the next test would return ``-(-I)^(2/3)``
         # which is less compact and elegant...
-        ("(-I)^(2/3)", "(-1) ^ (-1 / 3)", None),
+        #        ("(-I)^(2/3)", "(-1) ^ (-1 / 3)", None),
```

Comment:
Do I have it correct that this was working before, and not it is not?  _If_ that it the case, what's the insight here. (And if not, what is the insight here.)

Reply:
This works in #766 but not here. The issue comes from `Power.eval_check`.